### ISSUE TITLE
Title field panel customisations introduce invisible tab stops

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#9029]
In https://github.com/wagtail/wagtail/blob/9aeb2e3e49bb98201029d99fa613b0da9c6f1865/client/scss/components/forms/_title.scss#L18, we hide the title field panel’s header with `sr-only` so the field’s label is still present for screen reader users.

Unfortunately this isn’t correct – only the heading/label should be present. Right now, this approach also leads to invisible tab stops for the anchor links and section toggle/expand button.

We need to add extra styles to this title panel customisation so all anchor links and the button are `display: none`, so you can tab straight from the "Content" tab to the Title field.

**Comment by gasman** [comment]
Fixed in #9047
